Cargo.toml

```diff
@@ -36,11 +36,12 @@ bincode = "1"
 criterion = { version = "0.4.0", features = ["html_reports"] }
 hex = "0.4.2"
 rand = "0.8"
-rand_core = { version = "0.6", default-features = false }
+rand_core = { version = "0.6", default-features = false, features = ["getrandom"] }
 
 [[bench]]
 name = "dalek_benchmarks"
 harness = false
+required-features = ["rand_core"]
```

Comment:
@rozbb this is why it doesn't run benchmarks by default.

It's unrelated to this PR but I was getting a [build failure](https://github.com/dalek-cryptography/curve25519-dalek/actions/runs/3651302919/jobs/6168360815) without these changes.

Seems related to #447?

Reply:
#447 doesn't cause this in itself. `cargo bench` runs benches currently on `release/4.0`. Hmm

Reply:
Previously `std` was activated by default, and `std` activated `rand_core/std`.

The benchmarks [clearly do require `rand_core`](https://github.com/dalek-cryptography/curve25519-dalek/actions/runs/3651302919/jobs/6168360815), it's just it used to be "accidentally" enabled by default.

Reply:
Ah ok. Then I'll just make a note in the README for now